supertokens/logger.go

```diff
@@ -0,0 +1,29 @@
+package supertokens
+
```

Comment:
Add a comment showing how the log will look like (just like how it's done in the node SDK)